web-client/src/views/Public/OpinionPamphlets.tsx

```diff
@@ -0,0 +1,203 @@
+import { BigHeader } from '../BigHeader';
+import { Button } from '../../ustc-ui/Button/Button';
+import { Mobile, NonMobile } from '../../ustc-ui/Responsive/Responsive';
+import { connect } from '@cerebral/react';
+import { sequences, state } from 'cerebral';
+import React from 'react';
+
+export const OpinionPamphlets = connect(
+  {
+    openCaseDocumentDownloadUrlSequence:
+      sequences.openCaseDocumentDownloadUrlSequence,
+    opinionPamphletsHelper: state.opinionPamphletsHelper,
+  },
+  function OpinionPamphlets({
+    openCaseDocumentDownloadUrlSequence,
+    opinionPamphletsHelper,
+  }) {
+    return (
+      <>
+        <BigHeader text="United States Tax Court Reports" />
+
+        <NonMobile>
+          <section className="usa-section grid-container opinion-pamphlets">
```

Comment:
Great use of the USWDS utility and custom named classes in here, it's clear what each applied style is doing and why!